semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -604,6 +604,13 @@ trait TextDocumentOps { self: SemanticdbOps =>
               case _ =>
             }
 
+            def isClassOf(pos: m.Position): Boolean = {
+              val (chars, classOfChars) = (pos.input.chars, "classOf".toCharArray)
+
+              chars.length >= (classOfChars.length + pos.start) && (0 until classOfChars.length)
```

Comment:
Nit: please insert manual line breaks around infix operators to make it clearer to read the conditions 
```suggestion
              chars.length >= (classOfChars.length + pos.start) &&
                (0 until classOfChars.length)
```

Reply:
Fixed.